Dockerfile

```diff
@@ -0,0 +1,53 @@
+FROM lukemathwalker/cargo-chef:latest-rust-1.74-bookworm AS chef
+WORKDIR /app
+
+# Plan for dap-random-client
+FROM chef AS planner-client
+COPY . .
+RUN cargo chef prepare --recipe-path recipe.json
+
+# Build dap-random-client
+FROM chef AS builder-client
+COPY --from=planner-client /app/recipe.json recipe.json
+# Build dependencies - this is the caching Docker layer!
+RUN cargo chef cook --release --recipe-path recipe.json
+COPY . .
+RUN cargo build --release
+
+# Plan for janus-collector
+FROM chef AS planner-janus
+WORKDIR /app
+ARG JANUS_VERSION=0.6.6
```

Comment:
This defaults to the current version, we can also pass in this arg when building the image.